CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Disable resource link span ingestion. ([#2647](https://github.com/getsentry/relay/pull/2647))
 - Collect `http.decoded_response_content_length`. ([#2638](https://github.com/getsentry/relay/pull/2638))
 - Add TTID and TTFD tags to mobile spans. ([#2662](https://github.com/getsentry/relay/pull/2662))
+- Validate span timestamps and IDs in light normalization. ([#2679](https://github.com/getsentry/relay/pull/2679))
```

Comment:
Do we actually need a changelog for this? Span validation was already part of light normalization (through the `TransactionProcessor`), so that does not change from an outsider's point of view. Let me know if I'm missing something.

Reply:
Good catch! These checks didn't run when `renormalize` was enabled, which now do (first note on the PR description). I updated the changelog to be a bit more explicit.